db/migrations/postgres/000046_create_users.down.sql

```diff
@@ -1,5 +1,5 @@
 CREATE INDEX IF NOT EXISTS idx_users_email ON users (email);
 
-ALTER TABLE users DROP COLUMN IF NOT EXISTS remoteid;
+ALTER TABLE users DROP COLUMN IF EXISTS remoteid;
```

Comment:
Wow this was a bad bug :/

Reply:
Indeed it was, luckily morph caught these :)